docs/diagrams/ModelClassDiagram.puml

```diff
@@ -36,11 +38,14 @@ UserPrefs .up.|> ReadOnlyUserPrefs
 
 AddressBook *--> "1" UniquePersonList
 UniquePersonList --> "~* all" Person
-Person *--> Name
-Person *--> Phone
-Person *--> Email
-Person *--> Address
-Person *--> "*" Tag
+Person *--> "1" Name
+Person *--> "*" Phone
+Person *--> "*" Email
+Person *--> "*" Address
+Person *--> "0..1" Company
+Person *--> "*" Pronoun
+Person *--> "0..1" Job
+Person --> "*" Tag
```

Comment:
Why did you leave out Id? Or are you planning to include it in the description somewhere?

Reply:
I'm adding it to the DG rn - initially I had left it out because a simple, direct association to Person seemed more intuitive